src/app/screens/Library/index.tsx

```diff
@@ -1,16 +1,41 @@
-import React from 'react';
 import BookItem from './components/BookItem';
+import React from 'react';
+import {FlatList, View} from 'react-native';
+import {LibraryData} from '../../../constants/constants';
+import LibrarySeparator from './components/LibrarySeparator';
+import styles from './styles';
 
+interface dataItem {
+  item: {
+    id: number;
+    author: string;
+    title: string;
+    genre: string;
+    publisher: string;
+    year: string;
+    image_url: string | null;
+  };
+}
 const Library = () => {
-  return (
+  const renderItem = ({item}: dataItem) => (
```

Comment:
you could extract only the object items you need after defining the parameter

Reply:
Done.